net/pfSense-pkg-freeradius2/Makefile

```diff
@@ -56,6 +59,10 @@ do-install:
 		${STAGEDIR}${PREFIX}/pkg
 	${INSTALL_DATA} ${FILESDIR}${DATADIR}/info.xml \
 		${STAGEDIR}${DATADIR}
+	${INSTALL_DATA} ${FILESDIR}${PREFIX}/pkg/googleauth \
+		${STAGEDIR}/usr/local/etc/raddb/modules
+	${INSTALL_SCRIPT} ${FILESDIR}${PREFIX}/pkg/googleauth.py \
+		${STAGEDIR}/usr/local/etc/raddb/scripts
```

Comment:
No, you added 2 new files on FreeBSD-ports repository under `net/pfSense-pkg-freeradius2/files/usr/local/pkg` and on Makefile you are installing them to `STAGEDIR/usr/local/etc/raddb/scripts`. We try to keep the same directory structure under files/ that is used to install it